.envrc

```diff
@@ -0,0 +1,3 @@
+# Use `path:` syntax to avoid copying the entire repo to the Nix Store.
```

Comment:
Can you add a comment to what the `.envrc` file is and link to some docs? For those who don't know anything about nix like me 😅